src/main/java/appeng/me/cache/P2PCache.java

```diff
@@ -20,8 +20,10 @@
 
 
 import java.util.HashMap;
+import java.util.Random;
 
 import com.google.common.collect.LinkedHashMultimap;
+import com.google.common.collect.Maps;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.google.common.collect.Maps'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AUselessImportCheck)